content/_data/authors/olblak.adoc

```diff
@@ -2,4 +2,8 @@
 name: "Olivier Vernin"
 twitter: 0lblak
 github: olblak
+irc: olblak
 ---
+Olivier is the Jenkins infrastructure officer and Senior Operation Engineer at Cloudbees.
+As a regular contributor to the Jenkins infrastructure projects, he works on a wide range of tasks from services reliability to applications maintenance.
+Olivier is an infra coder who only believes in Git the mighty.
```

Comment:
Taking the recent code of conduct escalations, I would suggest rewording it to avoid offending people with other beliefs (ones who get offended by such things instead of considering it as a joke and not worrying)

Reply:
tbh I find it a bit excessive, but it always a matter of perspective, so I fine to remove the last line